task/sql-tasks.js

```diff
@@ -247,7 +388,17 @@ async function task_1_18(db) {
  *
  */
 async function task_1_19(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT
+        Customers.CustomerID,
+        CompanyName,
+        sum(UnitPrice * Quantity) AS 'TotalOrdersAmount, $'
+    FROM Customers JOIN Orders ON Customers.CustomerID = Orders.CustomerID JOIN OrderDetails ON Orders.OrderID = OrderDetails.OrderID
+    GROUP BY Customers.CustomerID
+    HAVING sum(UnitPrice * Quantity) > 10000
+    ORDER BY sum(UnitPrice * Quantity) DESC, CustomerID;

```

Comment:
Please, simplify ORDER BY and HAVING clauses using aliases from the prevuous step (don't forget escape quotes in sql query in node code)